src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,40 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.ValidationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LENGTH = 6;
+    private static final int MIN_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
+        if (user.getLogin() == null || user.getPassword() == null
+                || user.getAge() == null) {
+            throw new ValidationException("The value must not be a null");
+        }
```

Comment:
This is a code quality issue. You should split your checks into separate `if` statements, as it's recommended in the task checklist. In the current implementation, if some of the fields are `null`, you only inform the user about `null` fields, but you don't specify which field is `null`. You should check each field separately and throw an exception with an informative message if the field is `null`.